CDTDatastore/HTTP/CDTSessionCookieInterceptorBase.m

```diff
@@ -78,6 +78,22 @@ - (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorCon
         // Clear the cookies as we are no longer authorized
         _cookies = nil;
         context.shouldRetry = YES;
+    } else if (context.response.statusCode == 403) {
+        NSError *error = nil;
+        NSDictionary *statusCodeMessage = [NSJSONSerialization JSONObjectWithData:context.responseData
+                                                                        options:0
+                                                                          error:&error];
+        if(!error) {
+            NSString *http403Error = [[NSString alloc] init];
```

Comment:
You don't need to allocate this string object, because what you're doing is creating an object and then immediately throwing it away by assigning the pointer to the return value of `statusCodeMessage objectForKey` on the next line.

Just do `NSString *http403Error = [statusCodeMessage objectForKey:@"error"];`

Reply:
Fixed in 9b2f6bb.